packages/types/src/interfaces/staking/definitions.ts

```diff
@@ -10,5 +10,6 @@ export default {
         Account: 'AccountId',
       },
     },
+    VecDeque: 'Vec<any>',
```

Comment:
Do we have any other Vec type defined that it may collide with?

Reply:
Check this https://github.com/cennznet/api-types/pull/14#discussion_r517693253
VecDeque should be generic so that can be used for all the types

Reply:
updated and remove it from here.